compiler/luci/partition/include/luci/Partition.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_PARTITION_H__
+#define __LUCI_PARTITION_H__
+
+#include <luci/IR/Module.h>
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace luci
+{
+
+/**
+ * @brief PartitionTable holds partition information
+ */
+struct PartitionTable
+{
+  std::vector<std::string> groups;
+  std::string def_group;
+
+  // assign by opcode: OPCODE=group
+  std::unordered_map<std::string /* OPCODE */, std::string /* group */> byopcodes;
```

Comment:
Isn't it OpCodeName?
AFAIK, opcode is number...

Reply:
in the schema, there is `opcode_index:uint;` but not `opcode`.
agree on `OPCODE` is not exactly correct but I think the meaning is provided...

Reply:
`CircleOpcode.cpp` has following codes.
```cpp
std::string opcode_name(const CircleNode *node)
{
  QueryOpCode qoc;
  auto opcode = node->accept(&qoc);
  auto name = circle::EnumNameBuiltinOperator(opcode);
  return std::string(name);
}
```

And after I saw this, I thought that `OPCODE` is not string...
If `OPCODE` is used as string here, inconsistency maybe occurred...